test/scenarios/attestation/output/src/attestation/azext_attestation/tests/cmdlet/test_negative.py

```diff
@@ -19,8 +19,8 @@ class NegativeTest(ScenarioTest):
     def __init__(self, *args, **kwargs):
         super(NegativeTest, self).__init__(*args, **kwargs)
 
-    # EXAMPLE: /Operation/put/AttestationProviders_Create
     def test_create_provider(self):
+        # From /Operation/put/AttestationProviders_Create
```

Comment:
Is this change because of CLI Team's requirement ?

Reply:
No, after adding max&min mock test, it will be common that there are multiple examples for one single cmdlet. So each test case may contain multiple examples.
So the example name should be a comment for/ahead of the ```self.cmd(...)```, not for a test case function.